### ISSUE TITLE
Feature Request - Media field retrieve the url on  repeater field

### ISSUE THREAD

**Issue by bormedia** [Crocoblock/suggestions#825]
The goal was to create buttons / links that point to a downloadable file (PDF, XLS, etc.). In the end I set it up a "repeater" field, and requested to the user the label and the url from de file (manually they will have to upload it to the multimedia library and retrieve the url). It would been great to be able to upload the file directly on the meta field to the multimedia library and get the url automatically.

In the front, I am using the "Dynamic Repeater" widget (with a <a> tag to show the label and the url of the file) ... but when I bring the value of the "Media" field (on repeater field), it shows the value of the post code and not the URL of the media.

**Comment by pierrehansen** [comment]
Yes, I really need that too! In this scenario (which occurs quite often) we don't need any ID being added, we need the URL of the file!

Would be great if this option could be added.

**Comment by Crocoblock** [comment]
@pierrehansen @bormedia Please, use this macro https://crocoblock.com/knowledge-base/jetengine/macros-guide/#:~:text=meta%20key%20%E2%80%9Ctext%E2%80%9D.)-,%25media%2Dfield%7Cimg_url_by_id%25,-%E2%80%94%20insert%20this%20macro

If you have some other questions, please contact our support team https://crocoblock.com/help-center
